backend/db/firestore_db.js

```diff
@@ -0,0 +1,79 @@
+const { admin } = require('../db/admin');
+const db = admin.firestore();
+const collections = { sandbox: 'user_sandboxes' };
+const api404Error = require('../error/api404Error');
+
+const getCollectionDocs = async (collection) => {
+  try {
+    const booksRef = db.collection(collection);
+
+    booksRef.get().then((snapshot) => {
+      const data = snapshot.docs.map((doc) => ({
+        id: doc.id,
+        ...doc.data()
+      }));
+      console.log(`All docs in ${collection} collection`, data);
```

Comment:
Try removing logs that are not necessary. It might just bloat the logs.

Reply:
done https://github.com/armory-uci/vital/pull/14/commits/9f91935dcba3540c9f736fa025ed414e174b0f99